cloudsql.tf

```diff
@@ -0,0 +1,53 @@
+resource "google_sql_database_instance" "db-instance" {
+  region = "${var.region}"
+  project = "${var.project_id}"
+
+  settings {
+    tier = "db-n1-standard-2"
+    disk_autoresize = "true"
+
+    ip_configuration {
+      ipv4_enabled = "true"
+
+      authorized_networks = [
+		authorized_network_0 = "${module.terraform-gcp-natgateway.nat-gateway-ips["0"]}"
+		authorized_network_1 = "${module.terraform-gcp-natgateway.nat-gateway-ips["1"]}"
```

Comment:
Because we only have one NAT gateway instance, you can remove authorized_network_1 and authorized_network_2.